build.sbt

```diff
@@ -202,6 +205,25 @@ def currentGitHash(dir: File): String = {
   ).!!.trim
 }
 
+lazy val setupHail = taskKey[Unit]("Set up Hail")
```

Comment:
nit: might be better to call this `installHail`

Reply:
Also, I think we talked about having a separate test key for tests that rely on Hail? We should make sure that we don't require Hail or Hail deps for core Glow code.

Reply:
I've broken this out into `hail/test`, which runs a version of `pytest` but within the `hail` conda environment. We'll still have to streamline how this fits into releases; if we're trying to separate it from core Glow code, it makes sense to keep it out of `root` and make a new cross-release step to install/test/uninstall Hail.